lib/routes/dtfb.js

```diff
@@ -0,0 +1,25 @@
+const router = require('express').Router();
+const fetch = require('node-fetch');
+
+router.get('/tournaments', async (request, response) => {
+  try {
+    const result = await fetch('https://dtfb.de/index.php/termine');
+    const html = await result.text();
+    const name = html.match(/(?!=wrap>\s)(.*)(?=\s<br)/g);
```

Comment:
```suggestion
    const names = html.match(/(?!=wrap>\s)(.*)(?=\s<br)/g);
```
or 
```suggestion
    const torunamentNames = html.match(/(?!=wrap>\s)(.*)(?=\s<br)/g);
```

Reply:
It looks like you have to filter and trim the names for some reason.
Maybe there is a small issue in your RegExp.
For example:
```suggestion
    const name = html.match(/(?!=wrap>\s)(.+)(?=\s<br)/g);
```

`*` -> 0 or more
`+` -> 1 or more